tools/storybook/webpack.config.js

```diff
@@ -104,7 +105,7 @@ module.exports = ( storybookConfig ) => {
 				{
 					from: path.resolve(
 						__dirname,
-						`../../plugins/woocommerce/assets/client/admin/app/*.css`
+						`../../plugins/woocommerce/client/admin/build/app/*.css`
```

Comment:
Target the `build` css directly, as we have the `admin-library` as a dependency.